### PR TITLE
Updated to use 115200 as baudrate, corrected extruder steps and added README

### PR DESCRIPTION
**Pull request by perivar**
Updated to use 115200 as baudrate, corrected extruder steps and added README.

The double baudrate was too fast and the extruder steps wrong.

### CODE REVIEW

File: config/examples/Zonestar/P802M/Configuration.h [hunk 1]
```diff
@@ -21,8 +21,6 @@
  */
 #pragma once
 
-#define CONFIG_EXAMPLES_DIR "Zonestar/P802M"
-
```
Comment (thisiskeithb) [inline config/examples/Zonestar/P802M/Configuration.h]: These lines should remain since all configs have this define now.

Comment (perivar) [reply config/examples/Zonestar/P802M/Configuration.h]: I'm not an git expert. For some reason the changes I made to the ENDER 3 V2 config to add BLTouch got included in this pull request as well. I did however fix the missing CONFIG_... define.
Is it possible to only approve the Zonestar changes and Ignore the Ender 3 V2 changes?

Comment (thisiskeithb) [reply config/examples/Zonestar/P802M/Configuration.h]: You'll need to revert the changes you don't want to include or you can close this PR and start a clean one without the other changes.

**Comment by perivar** [comment]
Closing and making new PR with only the Zonestar changes.

**Pull request closed without merging**